src/query/QueryManager.js

```diff
@@ -2,15 +2,23 @@ import ObjectSelector from './ObjectSelector';
 import SwisstopoIdentify from './SwisstopoIdentify';
 import ScreenSpaceEventType from 'cesium/Source/Core/ScreenSpaceEventType';
 import i18next from 'i18next';
+import {OBJECT_HIGHLIGHT_COLOR} from '../constants';
+import {lv95ToDegrees} from '../projection.js';
+import Entity from 'cesium/Source/DataSources/Entity';
+import Cartesian3 from 'cesium/Source/Core/Cartesian3';
 
 
 export default class QueryManager {
   constructor(viewer) {
     this.objectSelector = new ObjectSelector(viewer);
     this.swisstopoIndentify = new SwisstopoIdentify();
+    this.viewer = viewer;
     this.scene = viewer.scene;
     this.enabled = true;
+    this.highlightEntity = null;
     viewer.screenSpaceEventHandler.setInputAction(click => this.onclick(click), ScreenSpaceEventType.LEFT_CLICK);
+    document.querySelector('ngm-object-information')
+      .addEventListener('closed', () => this.unhiglight());
```

Comment:
I think we should not do global DOM queries in components. It breaks the encapsulation of the component which is going against the concept of components.

I suggest instead to create a "onclose" function attribute that must be called by the 'ngm-object-information" component when it closes. If you remember, it is what I had started doing in my previous PR and removed for the merging.